$NetBSD: patch-Makefile,v 1.2 2025/01/29 17:25:01 vins Exp $

* Avoid syntax error in Makefile.
* Allow pkgsrc to define its PKGCONFIG_DIR.

--- Makefile.orig	2025-01-03 09:15:23.000000000 +0000
+++ Makefile
@@ -45,15 +45,13 @@ hwdata.pc: hwdata.pc.in
 
 install: Makefile.inc hwdata.pc
 	mkdir -p -m 755 $(DESTDIR)$(datadir)/$(NAME)
-	for foo in $(IDFILES) ; do \
-		install -m 644 $$foo $(DESTDIR)$(datadir)/$(NAME) ;\
-	done
+	install -m 644 $(IDFILES) $(DESTDIR)$(datadir)/$(NAME)
 	@if [ "$(blacklist)" = true ]; then \
 		mkdir -p -m 755 $(DESTDIR)$(libdir)/modprobe.d ;\
 		install -m 644 -T blacklist.conf $(DESTDIR)$(libdir)/modprobe.d/dist-blacklist.conf ;\
 	fi;
-	mkdir -p -m 755 $(DESTDIR)$(datadir)/pkgconfig
-	install -m 644 hwdata.pc $(DESTDIR)$(datadir)/pkgconfig/
+	mkdir -p -m 755 $(DESTDIR)$(PKGCONFIG_DIR)
+	install -m 644 hwdata.pc $(DESTDIR)$(PKGCONFIG_DIR)
 
 commit:
 	git commit -vas ||:
